src/providers/MeetingProvider/docs/MeetingManager.stories.mdx

```diff
@@ -43,7 +45,7 @@ Accepts a config object of `logLevel`, `postLogConfig`, `simulcastEnabled`, `ena
 Creates a meeting session using the meeting and attendee data passed. It will attempt to select default devices for the user.
 
 ```typescript
-import { EventReporter } from 'amazon-chime-sdk-js';
+import { EventReporter } from 'amazon-chime-sdk-js'; import MeetingManager from "./MeetingManager"; import {MeetingManagerConfig} from "./index";
```

Comment:
Could you please check if this renders correctly in storybook UI?

nit: We use single quotes for imports.
nit: We import directly from 'amazon-chime-sdk-component-library-react' and no relative imports.

Reply:
Seem like my IDE auto insert theses :)